projects/ui/src/Apis/api-types.ts

```diff
@@ -108,6 +108,16 @@ export type App = {
   teamId: string;
 };
 
+export type ApiKey = {
```

Comment:
I thought we already had this.  Is this new or bringing it back?

Reply:
Yeah it's a throwback 😄 - we had this for the Gloo Mesh Gateway portal server.

With the new implementation of the Gloo Gateway portal server (the version we're moving to with these apps + teams features), it's slightly different. Luckily the code for it was mostly there, and all tested before, so I know that certain features like `copyToClipboard` should work across browsers.